cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceContext.java

```diff
@@ -47,6 +51,14 @@
 
   /**
    * Overrides the input configuration of this MapReduce job to use
```

Comment:
We are switching to using "MapReduce program" or simply "MapReduce" instead of "MapReduce job", as "job" is really a specific run of the program. I think "this MapReduce" would work for this class.